airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -215,40 +212,22 @@ def __init__(
         self,
         *,
         task_definition: str,
-        wait_for_completion: bool = True,
-        waiter_delay: int | None = None,
-        waiter_max_attempts: int | None = None,
+        wait_for_completion: bool = True,  # obsolete
```

Comment:
Should we also add a warning saying `wait_for_completion/waiter_delay/waiter_max_attempts` has no effect going forward and point to the documentation in AWS? And also, a deprecation warning assuming they will eventually be removed would be nice. WDYT?

Reply:
hmm yeah right otherwise users won't remove them